components/drop-zone/provider.js

```diff
@@ -48,6 +48,7 @@ class DropZoneProvider extends Component {
 		window.addEventListener( 'dragover', this.dragOverListener );
 		window.addEventListener( 'drop', this.onDrop );
 		window.addEventListener( 'mouseup', this.resetDragState );
+		this.container = findDOMNode( this );
```

Comment:
We should have suppressed this warning with a valid reason for disabling.

Aside: IMO we shouldn't have ESLint warnings; only errors.